go.mod

```diff
@@ -4,6 +4,8 @@ go 1.21
 
 require (
 	github.com/cockroachdb/errors v1.11.1
+	github.com/go-jose/go-jose v2.6.1+incompatible
```

Comment:
I assume the change to `plugins/rangerguard/user/claim_test.go` removes this.

Reply:
indeed